SpatialGDK/Source/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -585,7 +620,23 @@ void USpatialActorChannel::OnReserveEntityIdResponse(const Worker_ReserveEntityI
 	RegisterEntityId(EntityId);
 
 	// Register Actor with package map since we know what the entity id is.
-	NetDriver->PackageMap->ResolveEntityActor(Actor, EntityId, improbable::CreateOffsetMapFromActor(Actor));
+	FClassInfo* Info = NetDriver->TypebindingManager->FindClassInfoByClass(Actor->GetClass());
```

Comment:
As discussed offline, put this back into `SetChannelActor` as well as change `GetHandoverSubobjects` to get the subobject pointers from the properties.